mayastor-test/e2e/node_disconnect/iscsi_reject/node_disconnect_iscsi_reject_test.go

```diff
@@ -30,7 +30,7 @@ func TestNodeLoss(t *testing.T) {
 }
 
 var _ = Describe("Mayastor node loss test", func() {
-	It("should verify behaviour when a node becomes inaccessible", func() {
+	It("should verify iscsi nexus behaviour when a node becomes inaccessible", func() {
```

Comment:
I love how you rewritten this (and above) to be more specific. I with devops/sre hat on regard dropped connection as "default" behavior - something went wrong on the network - but sure REJECT happens when something is misconfigured - which I'm guessing we're testing here (also filename suggest this). 

Thus I'm suggesting being explicit here too:
```suggestion
	It("should verify iscsi nexus behaviour when a node becomes inaccessible (iptables REJECT)", func() {
```

Reply:
Yes, happy to make these changes in the description.